platforms/software/build-init/src/test/groovy/org/gradle/buildinit/plugins/internal/GradlePropertiesGeneratorTest.groovy

```diff
@@ -50,9 +50,10 @@ class GradlePropertiesGeneratorTest extends Specification {
         propertiesFile.file
         propertiesFile.text.contains('org.gradle.parallel=true')
         propertiesFile.text.contains('org.gradle.caching=true')
+        propertiesFile.text.contains('org.gradle.configuration-cache=true')
     }
 
-    def "doesn't generate gradle.properties file if not incubating"() {
+    def "generates gradle.properties file with CC only if not incubating"() {
```

Comment:
💅 The name of test makes me think that CC is only enabled if not using incubating features